### PR TITLE
chore: rename L1 and L2 to parent and child chain in inbox

### LINKED ISSUE #336: Update language in `/inbox/inbox.ts`

### PR DESCRIPTION
**Pull request by douglance**
closes #336

### CODE REVIEW

File: src/lib/inbox/inbox.ts [hunk 1]
```diff
@@ -42,12 +45,12 @@ type ForceInclusionParams = FetchedEvent<MessageDeliveredEvent> & {
   delayedAcc: string
 }
 
-type GasComponentsWithL2Part = {
+type GasComponentsWithChainPart = {
   gasEstimate: BigNumber
   gasEstimateForL1: BigNumber
```
Comment (spsjvc) [inline src/lib/inbox/inbox.ts:50]:
```suggestion
  gasEstimateForParentChain: BigNumber
```

File: src/lib/inbox/inbox.ts [hunk 2]
```diff
@@ -42,12 +45,12 @@ type ForceInclusionParams = FetchedEvent<MessageDeliveredEvent> & {
...
   baseFee: BigNumber
   l1BaseFeeEstimate: BigNumber
```
Comment (spsjvc) [inline src/lib/inbox/inbox.ts:52]:
```suggestion
  parentChainBaseFeeEstimate: BigNumber
```

File: src/lib/inbox/inbox.ts [hunk 3]
```diff
@@ -42,12 +45,12 @@ type ForceInclusionParams = FetchedEvent<MessageDeliveredEvent> & {
   delayedAcc: string
 }
 
-type GasComponentsWithL2Part = {
+type GasComponentsWithChainPart = {
```
Comment (spsjvc) [inline src/lib/inbox/inbox.ts:48]:
```suggestion
type GasComponentsWithChildChainPart = {
```

File: src/lib/inbox/inbox.ts [hunk 4]
```diff
@@ -114,32 +122,34 @@ export class InboxTools {
...
-   * are making a delayed inbox message which doesn't need l1 calldata
+   * are making a delayed inbox message which doesn't need parentChain calldata
    * gas fee part.
    */
   private async estimateArbitrumGas(
-    transactionl2Request: RequiredTransactionRequestType,
-    l2Provider: Provider
-  ): Promise<GasComponentsWithL2Part> {
+    childChainTransactionRequest: RequiredTransactionRequestType,
+    chainProvider: Provider
```
Comment (spsjvc) [inline src/lib/inbox/inbox.ts:130]:
```suggestion
    childChainProvider: Provider
```

File: src/lib/inbox/inbox.ts (hunk 1 above)
Comment (yahgwai) [reply src/lib/inbox/inbox.ts:50]: yeh, agree, prob best to have the same name as nodeinterface

**Review by yahgwai — APPROVED** [review]

**Review by spsjvc — APPROVED** [review]
lgtm

**Pull request merged**